apps/kyb-app/src/components/organisms/DynamicUI/TransitionListener/TransitionListener.tsx

```
@@ -11,37 +12,49 @@
     const currentPageIndex = pages.findIndex(page => page.stateName === state);
     const prevPageIndex = pages.findIndex(page => page.stateName === prevStateRef.current);
 
     if (currentPageIndex < prevPageIndex) {
-      onPrevious && onPrevious(helpersRef.current, prevStateRef.current, state);
+      onPrevious &&
+        onPrevious({ setElementCompleted: helpersRef.current }, prevStateRef.current, state);
     }
 
     if (currentPageIndex > prevPageIndex) {
-      onNext && onNext(helpersRef.current, prevStateRef.current, state);
+      onNext && onNext({ setElementCompleted: helpersRef.current }, prevStateRef.current, state);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)